app/views/followups/new.html.erb

```diff
@@ -46,7 +46,7 @@
         <%= _('When you receive the paper response, please help others find ' \
                  'out what it says:') %>
         <ul>
-          <% if AlaveteliConfiguration.enable_annotations %>
+          <% if Alaveteli::Features.active?(Alaveteli::Features::ANNOTATIONS) %>
```

Comment:
Would be nice to supply a helper method so that in a view we could use something like:

```erb
<% if feature_active?(:annotations) %>
```